src/main/java/io/nats/client/api/ConsumerConfiguration.java

```diff
@@ -175,7 +175,7 @@ public String toJson() {
         JsonUtils.addField(sb, DELIVER_SUBJECT, deliverSubject);
         JsonUtils.addField(sb, DELIVER_GROUP, deliverGroup);
         JsonUtils.addField(sb, DELIVER_POLICY, GetOrDefault(deliverPolicy).toString());
-        JsonUtils.addField(sb, OPT_START_SEQ, startSeq);
+        JsonUtils.addUnsigned(sb, OPT_START_SEQ, startSeq);
```

Comment:
add as unsigned writes if the value is greater than 0 versus addField writes it if it's greater or equal

Reply:
Naming of `addUnsigned` method seems confusing.  Is not 0 (zero) an unsigned integer?   Why would I expect a side-effect that 0 (zero) is ignored?